src/lib/import-export/export/types.ts

```diff
@@ -8,21 +8,23 @@ export interface ExportOptions {
 	phpVersion: string;
 	splitDatabaseDumpByTable?: boolean;
 	specificSelections?: {
-		plugins?: string[];
-		themes?: string[];
-		uploads?: string[];
+		paths?: string[];
 	};
```

Comment:
```suggestion
specificSelectionPaths?: string[];
```
Could we simplify this even further, like so?

Reply:
https://github.com/Automattic/studio/pull/1940/commits/b3db695fe4a22b92160a0120befb1764207e738f